cypress/integration/user_cant_sign_up_with_the_same_email_spec.js

```diff
@@ -0,0 +1,20 @@
+describe("Registration", () => {
+  it("A user can't sign up with the same email twice - it gets redirected to the error page", () => {
+
+
+    cy.visit("/users/new");
+    cy.get("#name").type("someone");
+    cy.get("#email").type("someone@example.com");
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+
+    cy.visit("/users/new");
+    cy.get("#name").type("someone");
+    cy.get("#email").type("someone@example.com");
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+
+    cy.url().should("include", "/users");
```

Comment:
For me this line fails in the Cypress tests (this is the only spec file that fails out of all 6).

Error message from Cypress results:

_AssertonError: Timed out retrying after 4000ms: expected 'http://localhost:3030/sessions/new' to include '/users'_

Also tried signing up with existing email in browser using test server and it does just redirect me back to _/sessions/new_ and didn't spot anything odd in dev console.

We can check if it's same for anyone else.

Reply:
RESOLVED